tests/aiohttp/test_aiohttp_simple_api.py

```diff
@@ -61,6 +63,22 @@ def test_swagger_json(aiohttp_api_spec_dir, aiohttp_client):
     assert api.specification.raw == json.loads(json_)
 
 
+@asyncio.coroutine
+def test_swagger_yaml(aiohttp_api_spec_dir, aiohttp_client):
+    """ Verify the swagger.yaml file is returned for default setting passed to app. """
+    app = AioHttpApp(__name__, port=5001,
+                     specification_dir=aiohttp_api_spec_dir,
+                     debug=True)
+    api = app.add_api('swagger_simple.yaml')
+
+    app_client = yield from aiohttp_client(app.app)
+    spec_response = yield from app_client.get('/v1.0/swagger.yaml')
+    data_ = yield from spec_response.read()
+
+    assert spec_response.status == 200
+    assert api.specification.raw == yaml.load(data_)
```

Comment:
Codacy is complaining that this should be `safe_load(data_)`

Reply:
https://github.com/yaml/pyyaml/blob/d3eb7daf88e29bc20ffd4cfb833da3c49c617625/lib/yaml/__init__.py#L77